### ISSUE TITLE
`shub deploy` works but `shub schedule [spider]` fails for "default" target

### ISSUE THREAD

**Issue by umrashrf** [scrapinghub/shub#142]
`shub schedule` works for this:
```
[deploy]
url = https://dash.scrapinghub.com/api/scrapyd/
project = 1234
```

`shub schedule` fails for this:
```
[deploy:default]
url = https://dash.scrapinghub.com/api/scrapyd/
project = 1234
```

Whereas `shub deploy` works for both.

**Comment by jdemaeyer** [comment]
Is this for 1.5 (currently on PyPI) or for the master branch?

If the former, try the new shub 2.0, which includes major updates and which we'll release today: `pip install git+https://github.com/scrapinghub/shub.git`

**Comment by umrashrf** [comment]
Shub 1.5.0

**Comment by jdemaeyer** [comment]
I confirmed it's fixed in 2.0. We're currently waiting for the pip devs to release 8.0.1 and then the new all-shiny shub is gonna be released onto the world :)
